addons/event/models/event_event.py

```diff
@@ -644,6 +646,13 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
             for attendee in event.registration_ids.filtered(filter_func):
                 self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    def _get_event_description(self):
+        """ Returns the event description with a link to the event registration page. """
```

Comment:
Sugg:

```py
""" Description & URL of the event shortened to maximum 2000 characters.
Meant to be used for external content (ics/icalc). """
```

Would also be nice to justify the 2000 characters limit somehow, by mentioning external provider or browsers documentations of such limits for example.

Reply:
Hello @awa-odoo 
I wasn't able to find the official documentation of this URL limit so I have put the documentation from which I have taken the reference